CHANGELOG.md

```diff
@@ -1,13 +1,14 @@
 # Event Query Language - Changelog
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-
 # Version 0.9.12
-_Released 2021-XX-XX_
+
+ _Released 2022-03-15_
 
 ### Added
-* Support for `?` [optional field syntax](https://www.elastic.co/guide/en/elasticsearch/reference/current/eql-syntax.html#eql-syntax-optional-fields)
 
+* Support for `?` [optional field syntax](https://www.elastic.co/guide/en/elasticsearch/reference/current/eql-syntax.html#eql-syntax-optional-fields) if the Elasticsearch flag is enabled
+* Support for [repeated sequences](https://www.elastic.co/guide/en/elasticsearch/reference/current/eql-syntax.html#eql-sequences) if the Elasticsearch flag is enabled
```

Comment:
I don't see any specific reference to the repeated syntax in the docs - we can ping the team for updates

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/current/eql-syntax.html#eql-with-runs-statement

Reply:
woot

```suggestion
* Support for [repeated sequences](https://www.elastic.co/guide/en/elasticsearch/reference/current/eql-syntax.html#eql-with-runs-statement) if the Elasticsearch flag is enabled
```